appengine/flexible/api_gateway/README.md

```diff
@@ -0,0 +1,80 @@
+# Python Google Cloud Microservices Example - API Gateway
+
+This example demonstrates how to deploy multiple python services to [App Engine Flexible Environment](https://cloud.google.com/appengine/docs/flexible/)
+
+## To Run Locally
+
+1. You will need to install Python 3 on your local machine
+
+2. Install virtualenv
+```Bash
+$ pip install virtualenv
+```
+
+3. To setup the environment in each server's directory:
+```Bash
+$ virtualenv -p python3 env
+$ source env/bin/activate
+$ pip install -r requirements.txt
+$ deactivate
+```
+
+4. Change the url's in the API gateway `gateway/api_gateway.py` to the
+development ports specified in line `gateway/api_gateway.py:6`
+```Python
+development ports {static: 8003, flask1: 8001, flask2: 8002}
+```
+
+5. To run each server locally:
+```Bash
+$ source env/bin/activate
+$ export FLASK_DEBUG=1 #optional to run in debug mode
+$ python <server name>.py development
+$ deactivate #after shutting down server
+```
+
+## To Deploy to App Engine
+
+### YAML Files
+
+Each directory contains an `app.yaml file`.  This file will be the same as if
+each service were its own project except for the `service` line. For the gateway:
+```YAML
```

Comment:
Just link to the files, no need to include them here. (They'll inevitably get out of sync)